### PR TITLE
Fixes airlock electronic ID-related shitcode

### LINKED ISSUE #11117: Airlock electronics access not very intuitive.
(WEB REPORT BY: zalo1302 REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 98ae8bc689f3002352e409832e2daf087c686c7d 
> 
> General description of the issue
> The procedure to set access requirement to airlock electronics it's not very intuitive.
> 
> What you expected to happen
> Have my ID in PDA wearing/in my hand or have ID in hand to swipe. 
> 
> What actually happened
> I have to put the ID out of the PDA AND wear it to swipe the airlock electronic.
> 
> Steps to reproduce if possible
> Just try with any airlock electronic.

Also i don't know if there are other electronic where this same procedure is used. It would be nice if it worked in all cases.

### PR DESCRIPTION
**Pull request by 9600bauds**
Fixes #11117

### CODE REVIEW

File: code/game/machinery/doors/airlock_electronics.dm [hunk 1]
```diff
@@ -89,28 +89,20 @@
...
 	if(href_list["close"])
 		usr << browse(null, "window=airlock")
 		return
 
 	if(href_list["login"])
 		if(ishuman(usr))
-			var/mob/living/carbon/human/H=usr
-			var/obj/item/I = usr.get_active_hand()
-			if(!istype(I, /obj/item/weapon/card) || !istype(I, /obj/item/device/pda))
-				I = H.wear_id
```
Comment (9600bauds) [inline code/game/machinery/doors/airlock_electronics.dm]: n i c e
c o p y p a s t e

File: code/game/machinery/doors/airlock_electronics.dm [hunk 2]
```diff
@@ -89,28 +89,20 @@
 
 /obj/item/weapon/circuitboard/airlock/Topic(href, href_list)
 	if(..()) return 1 //Its not as though this does ANYTHING
-	if(!Adjacent(usr) || usr.stat || usr.restrained() || (!ishuman(usr) && !isrobot(usr)) || icon_state == "door_electronics_smoked" || installed)
+	if(!Adjacent(usr) || usr.restrained() || (!ishuman(usr) && !isrobot(usr)) || icon_state == "door_electronics_smoked" || installed)
```
Comment (ComicIronic) [inline code/game/machinery/doors/airlock_electronics.dm]: Why remove the stat check?

Comment (9600bauds) [inline code/game/machinery/doors/airlock_electronics.dm]: Restrained already does it
Checking .stat directly is bad anyways, you should check for isUnconscious() or something like that

Comment (ComicIronic) [inline code/game/machinery/doors/airlock_electronics.dm]: That's a weird check for restrained(), shouldn't it be under incapacitated() or whatever?

Comment (9600bauds) [inline code/game/machinery/doors/airlock_electronics.dm]: oh I'm stupid, I got incapacitated() and restrained() mixed up

**Pull request merged**